cves/CVE-2015-0220.yml

```diff
@@ -49,7 +49,18 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Django provided a method to attempt to combat issues with urls to prove they
+  are safe, but it failed in this instance as this CVE allowed malicious users
+  to manipulate urls (opening up cross-site scripting vulnerabilities) via
+  changing the leading whitespace before the url's actual location. Once
+  changed, the method would fail to catch urls that were unsafe due to the
+  whitespace, and the url would be processed in the system.
+
+  Essentially, when the method was run with a string like "\nthiswebsite.com"
+  it validated to true. But that whitespace could allow someone to do something
```

Comment:
You are using the word "it" to refer to the method, might be worth replacing "it" with a more specific term like "the method" for clarity.

Reply:
Additionally replace someone with "bad actor", to show intent of the user.